### DISCUSSION TITLE
just found this... love it

### DISCUSSION THREAD

**Discussion by dkguy** [bnpr/Malt#591] [General]
hey found this render engine yestoday, is there any way to support it

**Comment by Psi25** [comment]
idk if theres a donation page. there probably is and im just blind

**Comment by pragma37** [comment]
Hi @dkguy, thank you!

Just do cool stuff and share it with others.
If you can code, contributions are super-welcome.

I used to have a Patreon, but I had to close it once I started working for the Blender Foundation.
